tdrs-backend/tdpservice/settings/common.py

```diff
@@ -175,12 +151,23 @@ class Common(Configuration):
         "django.contrib.staticfiles.finders.AppDirectoriesFinder",
     )
 
+    # By default collectstatic will store files locally so these settings are
+    # not used but they must be defined, lest the server will fail to startup.
+    AWS_S3_STATICFILES_ACCESS_KEY = None
```

Comment:
Note: In the future we may want to support serving staticfiles over localstack for testing purposes or using a live S3 to serve staticfiles from a local environment. This would need to be updated for that.